.example.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_GRAPHQL_API_URL=http://localhost:4000/
+REACT_APP_MCP_DATA_MAPPING_UTIL=http://localhost:5000/
```

Comment:
Since this is a frontend and it won't ever contain secrets, `.env` is in version control, so we can just include this in `.env` instead of having an example file

Reply:
I'm hesitant to do that in case things change... @puckybreg, ruling?